Production_scripts/Process_extracts/extract_gp_ooh_diagnosis.R

```diff
@@ -0,0 +1,143 @@
+#####################################################
+# Draft pre processing code for Gp Out of Hours - Diagnosis
+# Author: Jennifer Thom
+# Date: April 2022
+# Written on RStudio Server
+# Version of R - 3.6.1
+# Input - GP-OoH-
+#         # consultations-extract-.csv
+#         # diagnosis-extract-.csv
+#         # outcomes-extract-.csv
+#
+# Description - Preprocessing of GP out of hours raw BOXI file.
+#              Tidy up file in line with SLF format
+#              prior to processing.
+#####################################################
+
+# Load Packages
+library(readr)
+library(dplyr)
+library(tidyverse)
+library(createslf)
+library(stringr)
+
+
+## Load Read code lookup----------------------------
+
+## TODO - Put into function##
+read_code_lookup <- haven::read_sav(
+  get_file_path(
+    directory = fs::path(get_slf_dir(), "Lookups"),
+    file_name = "ReadCodeLookup.zsav"
+  )
+) %>%
+  rename(
+    readcode = "ReadCode",
+    description = "Description"
+  )
+
+## Load extract file---------------------------------
+
+year <- "1920"
+
+diagnosis_file <- read_csv(
+  file = get_boxi_extract_path(year, "GP_OoH-d"),
+  col_type = cols(
+    `GUID` = col_character(),
+    `Diagnosis Code` = col_character(),
+    `Diagnosis Description` = col_character()
+  )
+) %>%
+  # rename variables
+  rename(
+    guid = `GUID`,
+    readcode = `Diagnosis Code`,
+    description = `Diagnosis Description`
+  )
+
+
+## Deal with Read Codes --------------------------
+
+diagnosis_clean <- diagnosis_file %>%
```

Comment:
```suggestion
diagnosis_read_codes <- diagnosis_file %>%
```

rename this and leave the `diagnosis_clean` for the next section dataframe name, I got an error so it allows sections to be tested separately